app/src/main/java/com/example/pomodorotimer/Timer.kt

```diff
@@ -81,5 +81,9 @@ class Timer{
     fun endTimer() {
         isCounting = false
         needResume = false
+        //reset timer
+        secondToCount=0
```

Comment:
May be resetting timer here is not necessary, because startTimer() will always load the worktime or breaktime

Reply:
It's more for orientation change after cancel. If it's not reset, it displays the nothing or the timer before cancel. But I can't use startTimer () because we don't want to start the timer if its cancelled, or can we? Maybe we can now after I add a if not counting condition. I'll test it again later

Reply:
I am worry that some thing might break, because both work timer and start timer will use this endtimer() method.

ok I get what you mean, so currently line 145 will always set the work state back to work when user cancer the timer. Because after cancel (no matter it was work or break previously), we always start a timer from work state.

Reply:
#26